### ISSUE TITLE
selectOneMenu: Component is skipped by screenreader in browse mode

### ISSUE THREAD

**Issue by miedrl** [primefaces/primefaces#8242]
**Describe the defect**
Screenreaders typically have a browse mode in which you can read the content of a page and a focus mode in which you can also interact with the page (e.g. clicking a button). In both modes each element should be readable by the screenreader. The defect here is that, when in browse mode, selectOneMenu components are skipped by the screenreader so a blind user would not know that they exist (the same problem may exist with other components such as selectCheckboxMenu but I did not try that).

**Environment:**
 - PF Version: _10.0_
 - tested with the screenreader NVDA

**To Reproduce**
1. Go to https://www.primefaces.org/diamond/input.xhtml
2. Scroll down to the "Listbox" panel
3. Use the browse mode of a screenreader (e.g. NVDA) to test if the Dropdown and CheckboxMenu are read by the screenreader.
4. See error: The headlines "Dropdown" and "CheckboxMenu" are read aloud but the select components are skipped

**Expected behavior**
For accessibility, select components like selectOneMenu or selectCheckboxMenu as well as their given options should be read by the screenreader, not skipped.

**Comment by melloware** [comment]
I would report this as a bug to NVDA.  Our SelectOne has this ` aria-owns="j_idt120_items" role="combobox" aria-haspopup="listbox" aria-expanded="false"` which is proper ARIA identification so I would check with NVDA and report the bug there.

**Comment by melloware** [comment]
As a caveat if you find something that NVDA reports we are not doing correctly for ARIA standards we are happy to fix but this generic report about 1 single reader does us no good.
